### PR TITLE
Adding konoha

### PR DESCRIPTION
**Pull request by BastianZim**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](t) before pinging a team.

_Created using Grayskull_

### CODE REVIEW

**Comment by BastianZim** [comment]
No review required yet, still need to fix build process.

**Comment by BastianZim** [comment]
@conda-forge/help-python Ready for review! 😊

File: recipes/konoha/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (dopplershift) [inline recipes/konoha/LICENSE:1]: `LICENSE` is already included in the tarball from PyPI, so you don't need it--go ahead and remove from the PR.

**Review by dopplershift — CHANGES_REQUESTED** [review]

File: recipes/konoha/LICENSE (hunk 1 above)
Comment (BastianZim) [reply recipes/konoha/LICENSE:1]: Just to confirm, should I remove the whole license section, or just this part?

Comment (BastianZim) [reply recipes/konoha/LICENSE]: Update: Removed file and left _meta.yaml_ unchanged

**Review by dopplershift — APPROVED** [review]

**Pull request merged**